src/document/generator.ml

```diff
@@ -560,21 +560,27 @@ struct
 
 
   let extension_constructor (t : Odoc_model.Lang.Extension.Constructor.t) =
-    (* TODO doc *)
-    let doc = constructor (t.id :> Paths.Identifier.t) t.args t.res in
-    O.documentedSrc (O.txt "| ") @ doc
+    let id = (t.id :> Paths.Identifier.t) in
+    match Url.from_identifier ~stop_before:true id with
+    | Error e -> failwith (Url.Error.to_string e)
+    | Ok url ->
+       let anchor = Some url in
+       let attrs = ["def"; url.kind] in
```

Comment:
Not sure if we want other `attrs` here. The variant case uses `def variant`, but I'm not sure it affects any of the renderers.

Reply:
Let's keep it so far, it's easy to change if we need more.